test/test_cases/run_3d_held_suarez.jl

```diff
@@ -0,0 +1,90 @@
+using OrdinaryDiffEq: SSPRK33
+using ClimaCorePlots, Plots
+using UnPack
+
+using CLIMAParameters
+using ClimaAtmos.Utils.InitialConditions: init_3d_baroclinic_wave
+using ClimaAtmos.Domains
+using ClimaAtmos.BoundaryConditions
+using ClimaAtmos.Models: ConstantViscosity
+using ClimaAtmos.Models.Nonhydrostatic3DModels
+using ClimaAtmos.Simulations
+
+# Set up parameters
+const FT = Float64
+const day = FT(86400)
+Base.@kwdef struct HeldSuarezParameters <: CLIMAParameters.AbstractEarthParameterSet 
+    k_a = FT(1 / (40 * day))
+    k_f = FT(1 / day)
+    k_s = FT(1 / (4 * day))
+    ΔT_y = FT(60)
+    Δθ_z = FT(10)
+    T_equator = FT(315)
+    T_min = FT(200)
+    σ_b = FT(7 / 10)
+end
+
+stepper = SSPRK33()
+nelements = (4, 10)
+npolynomial = 4
+dt = FT(5)
+
+params = HeldSuarezParameters()
+# function run_3d_baroclinic_wave(
+#     ::Type{FT};
+#     stepper = SSPRK33(),
+#     nelements = (6, 10),
+#     npolynomial = 3,
+#     case = :default,
+#     dt = 0.02,
+#     callbacks = (),
+#     test_mode = :regression,
+# ) where {FT}
+    
+
+    domain = SphericalShell(
+        FT,
+        radius = CLIMAParameters.Planet.planet_radius(params),
+        height = FT(30.0e3),
+        nelements = nelements,
+        npolynomial = npolynomial,
+    )
+
+    model = Nonhydrostatic3DModel(
+        domain = domain,
+        boundary_conditions = nothing,
+        parameters = params,
+        hyperdiffusivity = FT(1e16),
+        vertical_diffusion = ConstantViscosity(ν = FT(1)),
```

Comment:
Have we been running this vertical diffusion? We shouldn't. (1 m2/s is quite strong.)

Reply:
No, we didn't use it.